docs/topic/cloud-access.md

```diff
@@ -109,6 +109,14 @@ To do so, follow these steps:
    AWS accounts we create.
 4. Create the account! They'll receive an email with appropriate instructions.
 
+### Create new accounts in our SSO
+
+We often create new accounts in the 2i2c organization if we want to run community-specific infrastructure attached to that account (e.g., a dedicated Kubernetes cluster).
+To create a new Account under our AWS SSO, follow these steps:
+
+1. Do XXX @yuvipanda can you fill in these steps?
```

Comment:
@yuvipanda could you add in just a few steps here to point people in the right direction?

Reply:
Are these the instructions we need to follow? https://docs.aws.amazon.com/singlesignon/latest/userguide/step3.html

Reply:
@yuvipanda can you please confirm yes or no or link to page that has instructions on how to add an account to SSO? We don't need anything complex, just something to spread this knowledge a bit